crates/sgx_validation/src/config.rs

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use serde::{Deserialize, Deserializer, Serialize};
+use sgx::parameters::Features;
+use validation_common::Measurements;
+
+#[derive(Clone, Deserialize, Debug, Serialize)]
+pub enum SgxFeatures {
+    CET,
+    Debug,
+    EIntKey,
+    KSS,
+    ProvisioningKey,
+}
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+pub struct Config {
+    /// Values for `mrsigner` in the report body, as `Measurements::signer()`
+    /// This is the list of public keys which have signed the Enarx binary.
+    /// Values for `mrenclave` in the report body, as `Measurements::hash()`
+    /// This is the hash of the Enclave environment after the Enarx binary is loaded
+    /// but before any workload is loaded, so this is a hash of the Enarx binary
+    /// in memory.
+    #[serde(default, flatten)]
+    pub enarx: Measurements,
```

Comment:
enarx? why not `measurements` ... I would remove `enarx` from all keywords..

Reply:
Fixed